lib/image-diff.js

```diff
@@ -81,8 +81,8 @@ ImageDiff.createDiff = function (options, cb) {
     }
 
     // Callback with pass/fail
-    var totalDifference = resultInfo[1];
-    return cb(null, totalDifference === '0');
+    var totalDifference = resultInfo[2]; // this is the percentage difference
```

Comment:
We should be renaming this variable since it's now the percentage difference, not the total difference.